src/plugins/intel_gpu/src/graph/ctc_greedy_decoder.cpp

```diff
@@ -18,6 +20,34 @@ layout ctc_greedy_decoder_inst::calc_output_layout(ctc_greedy_decoder_node const
     return layout(output_type, input_node_layout.format, prim->output_tensor);
 }
 
+template<typename ShapeType>
+std::vector<layout> ctc_greedy_decoder_inst::calc_output_layouts(ctc_greedy_decoder_node const& /*node*/, const kernel_impl_params& impl_param) {
+    std::vector<layout> layouts;
+
+    auto desc = impl_param.typed_desc<ctc_greedy_decoder>();
+    // auto input_layout = impl_param.get_input_layout();
+
+    ov::op::v6::CTCGreedyDecoderSeqLen op;
```

Comment:
This calc_output_layouts is used for both op::v0 CTCGreedyDecoder and op::v6 CTCGreedyDecoderSeqLen. Please check on both ops.

Reply:
Have you checked dynamic shape single layer test of op::ov::v0::CTCGreedyDecoder? If no, please add the test case together, so you can check it.

Reply:
Added dynamic shape support for op::ov::v0::CTCGreedyDecoder and its functional tests. Thanks for your comments.